src/query/functions/src/scalars/datetime.rs

```diff
@@ -131,6 +136,125 @@ fn int64_domain_to_timestamp_domain<T: AsPrimitive<i64>>(
     })
 }
 
+fn register_convert_timezone(registry: &mut FunctionRegistry) {
+    // 2 arguments function [target_timezone, src_timestamp]
+    registry.register_passthrough_nullable_2_arg::<StringType, StringType, TimestampType, _, _>(
```

Comment:
The data type of `src_timestamp` should be `Timestamp`, so we don't have to parse the time string in this function.